save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/InitI18n.kt

```diff
@@ -37,7 +37,8 @@ fun initI18n(isDebug: Boolean = false, interpolationEscapeValue: Boolean = false
                     'vulnerability',
                     'comments',
                     'dates',
-                    'index'
+                    'index',
+                    'profile'
```

Comment:
can be added a `,` at the end to avoid change in last line when a new value is added

Reply:
Didnt know supported it by `js` native code or not, so add without `,`